scripts/import-graph-report.py

```diff
@@ -51,7 +51,7 @@ def compare_counts(base_file, head_file, changed_files_txt):
         messages.append(f'| {file} | {base_count} | {head_count} | {sign}{diff} ({sign}{percent:.2f}%) |')
 
     # Build the message
-    message = '### Import summary\n\n'
+    message = ''
     if messages:
         message += '<details><summary>Dependency changes</summary>\n\n'
```

Comment:
Here the `<details>` should now be removed, no?

Reply:
Removed!